### PR TITLE
clarify direct creator upload using tus

### PR DESCRIPTION
**Pull request by zaidf**

### CODE REVIEW

**Comment by zaidf** [comment]
@marciocloudflare feel free to review this and merge

File: products/stream/src/content/uploading-videos/direct-creator-uploads.md [hunk 1]
```diff
@@ -225,9 +225,15 @@ async function gatherResponse(response) {
 }
 ```
 
-Once you have the tokenized URL, you can pass it to the tus client to begin the upload. For details on using a tus client, refer to the [Resumable uploads with tus ](https://developers.cloudflare.com/stream/uploading-videos/upload-video-file#resumable-uploads-with-tus-for-large-files) article.
+Once you have an endpoint that returns the tokenized upload URL in the location header, you can use it by setting the tus client to make a request to *your* endpoint. For details on using a tus client, refer to the [Resumable uploads with tus ](https://developers.cloudflare.com/stream/uploading-videos/upload-video-file#resumable-uploads-with-tus-for-large-files) article.
```
Comment (third774) [inline products/stream/src/content/uploading-videos/direct-creator-uploads.md]: Two suggestions:
1. Should we put `Location` in backticks and capitalize to draw more attention and match the formatting from [this section](https://developers.cloudflare.com/stream/uploading-videos/direct-creator-uploads#using-tus-recommended-for-videos-over-200mb)?
2. Should we change "returns the tokenized upload URL _in_ the location header" to "returns the tokenized upload URL _from_ the location header"?

**Review by YoutacVA — APPROVED** [review]

File: products/stream/src/content/uploading-videos/direct-creator-uploads.md (hunk 1 above)
Comment (renandincer) [reply products/stream/src/content/uploading-videos/direct-creator-uploads.md]:
```suggestion
Once you have an endpoint that returns the tokenized upload URL from the `location` header, you can use it by setting the tus client to make a request to *your* endpoint. For details on using a tus client, refer to the [Resumable uploads with tus ](https://developers.cloudflare.com/stream/uploading-videos/upload-video-file#resumable-uploads-with-tus-for-large-files) article.
```

File: products/stream/src/content/uploading-videos/direct-creator-uploads.md [hunk 2]
```diff
@@ -225,9 +225,15 @@ async function gatherResponse(response) {
...
 
-To test your end point which returns the tokenized URL, visit the [tus codepen demo](https://codepen.io/cfzf/pen/wvGMRXe) and paste your end point URL in the "Upload endpoint" field.
+### Testing your Direct Creator Upload Endpoint
```
Comment (renandincer) [inline products/stream/src/content/uploading-videos/direct-creator-uploads.md:230]:
```suggestion
### Testing suggestions
```

File: products/stream/src/content/uploading-videos/direct-creator-uploads.md [hunk 3]
```diff
@@ -225,9 +225,15 @@ async function gatherResponse(response) {
...
+Once you have an endpoint that returns the tokenized upload URL from the `location` header, you can use it by setting the tus client to make a request to *your* endpoint. For details on using a tus client, refer to the [Resumable uploads with tus ](https://developers.cloudflare.com/stream/uploading-videos/upload-video-file#resumable-uploads-with-tus-for-large-files) article.
 
-To test your end point which returns the tokenized URL, visit the [tus codepen demo](https://codepen.io/cfzf/pen/wvGMRXe) and paste your end point URL in the "Upload endpoint" field.
+### Testing your Direct Creator Upload Endpoint
+
+Once you have built your endpoint which calls Stream and returns the tokenized URL in the location header, you can test it with this [tus codepen demo](https://codepen.io/cfzf/pen/wvGMRXe). In the demo codepen, paste your end point URL in the "Upload endpoint" field and then try to upload a video.
```
Comment (marciocloudflare) [inline products/stream/src/content/uploading-videos/direct-creator-uploads.md]: Change "Upload endpoint" to `Upload endpoint`.

File: products/stream/src/content/uploading-videos/direct-creator-uploads.md [hunk 4]
```diff
@@ -225,9 +225,15 @@ async function gatherResponse(response) {
...
 
-To test your end point which returns the tokenized URL, visit the [tus codepen demo](https://codepen.io/cfzf/pen/wvGMRXe) and paste your end point URL in the "Upload endpoint" field.
+### Testing your Direct Creator Upload Endpoint
+
+Once you have built your endpoint which calls Stream and returns the tokenized URL in the location header, you can test it with this [tus codepen demo](https://codepen.io/cfzf/pen/wvGMRXe). In the demo codepen, paste your end point URL in the "Upload endpoint" field and then try to upload a video. 
+
+When using Direct Creator Uploads, the "Upload endpoint" field in the demo should contain the url to your endpoint; not to the videodelivery.net tokenized URL. This is the most common reason Direct Creator Uploads fail using tus. Customers often set the tus url to the videodelivery.net URL instead of to their endpoint which *returns* the videodelivery.net URL.
```
Comment (marciocloudflare) [inline products/stream/src/content/uploading-videos/direct-creator-uploads.md]: Change "Upload endpoint" to `Upload endpoint` with `` (backticks).
Also, replace `;` with `,` after `to your endpoint`

**Comment by marciocloudflare** [comment]
Hey @zaidf. Could someone take a look at my comments? Thanks!

**Comment by KimJ15** [comment]
@sejoker would you be able to review and approve Márcio's comments? Thanks!

**Review by sejoker — APPROVED** [review]

**Comment by sejoker** [comment]
@KimJ15 done.

**Review by KimJ15 — APPROVED** [review]

**Pull request merged**